pwndbg/color/context.py

```diff
@@ -97,15 +98,28 @@ def format_flags(value, flags, last=None):
 
     names = []
     for name, bit in flags.items():
-        bit = 1 << bit
-        if value & bit:
-            name = name.upper()
-            name = flag_set(name)
+        # If the size is not specified, assume it's 1
+        if isinstance(bit, int):
+            size = 1
         else:
-            name = name.lower()
-            name = flag_unset(name)
+            assert len(bit) == 2
```

Comment:
Consider adding a msg, so we know what is the assertion triggered :D

Reply:
We should know from the stack trace, right?